pkgs/by-name/as/asmc/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "asmc";
+  version = "2.36.02";
+
+  src = fetchFromGitHub {
+    owner = "nidud";
+    repo = "asmc";
+    rev = "7dda9a180031c3723e270e0b6f51f4d8b73004a9";
+    hash = "sha256-Uk9K8m8r9stetIlUoVfBlayeDZSDvt1EmTI68W8dwso=";
+  };
+
+  makeFlags = [
+    "-C"
+    "source/asmc"
```

Comment:
Can't we change the directory prior to the build using `sourceRoot`?

Reply:
Yeah, probably. Previously `install` was copying headers from below that root so it felt more elegant to do it this way. I’m ambivalent. The need to sprinkle `${finalAttrs.src.name}` in `sourceRoot` always annoys me.